website/docs/r/security_center_hybrid_vm_server_vulnerability_assessment.html.markdown

```diff
@@ -0,0 +1,51 @@
+---
+subcategory: "Security Center"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_security_center_hyrbid_vm_server_vulnerability_assessment"
+description: |-
+  Manages an Azure Vulnerability Assessment (Qualys) to an Azure ARC server.
+---
+
+# azurerm_security_center_hyrbid_vm_server_vulnerability_assessment
+
+Manages an Azure Server Vulnerability Assessment (Qualys) to a VM.
+
+-> **NOTE** Azure Defender has to be enabled on the subscription in order for this resource to work.
+See this [documentation](https://docs.microsoft.com/en-us/azure/security-center/security-center-get-started) to get started.
```

Comment:
fwiw if we remove the language here it'll redirect to the users locale if available in that language

```suggestion
See this [documentation](https://docs.microsoft.com/azure/security-center/security-center-get-started) to get started.
```